app/javascript/dashboard/components/buttons/ResolveButton.vue

```diff
@@ -56,6 +56,10 @@ export default {
       this.$store.dispatch('toggleStatus', this.currentChat.id).then(() => {
         bus.$emit('newToastMessage', this.$t('CONVERSATION.CHANGE_STATUS'));
         this.isLoading = false;
+
+        if (this.currentChat.nextId && wootConstants.STATUS_TYPE.RESOLVED == this.currentChat.status) {
```

Comment:
Move this to to a variable for better readability ?
`const canOpenNextChat =  this.currentChat.nextId && wootConstants.STATUS_TYPE.RESOLVED == this.currentChat.status`

Reply:
https://github.com/chatwoot/chatwoot/pull/757/commits/02ba6d1375e47db167ad3df15d7900d381358947